File: download_gifs.js
var exec = require('child_process').exec;
var clips = require('./prelinger.json'); clips = clips.clips;

// var count = 0;

// var dl_thumb = function (t)  {
//     console.log("downloading", t);
//     exec('cd gifs/prelinger; curl -O ' + t);
//     count++;
// }


setInterval(function(){
    console.log('hit');
}, 2000);


// for (var i = 0; i < 1; i++) {
//     var t = clips[i]['thumbnail'].split("?")[0];
//     dl_thumb(t);
// }

File: download_gifs.py
import json
import os
from pprint import pprint
json_data = open('prelinger.json')

data = json.load(json_data)

for clip in data['clips']:
    thumb = clip['thumbnail'].rpartition('?')[0]
    print thumb
    os.system('cd gifs/prelinger/; curl -O ' + thumb)

json_data.close()
